DataConnectors/SalesforceServiceCloud/SalesforceServiceCloud_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "SalesforceServiceCloud",
+    "title": "Salesforce Service Cloud (Preview)",
```

Comment:
Let's keep this in the json file so that it's retained and we are aware of it for future. This is not automatically added in GitHub - it needs to be manually committed.

Reply:
Let's keep the title clean and remove  (Preview).  

It will be added in PROD based on the below flag value.

  "availability": {
        "status": 1,
        **"isPreview": true**
   }

Reply:
Done